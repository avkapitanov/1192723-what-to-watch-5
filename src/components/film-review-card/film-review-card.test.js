import React from "react";
import renderer from "react-test-renderer";
import FilmReviewCard from "./film-review-card";

const review = {
  id: 1,
  user: {
    id: 16,
    name: `Mollie`
  },
  rating: 3.3,
  comment: `I personally found this movie to be boring. Definitely one of the most boring movies I've ever seen.`,
  date: `2020-10-30T11:53:40.154Z`
};

it(`FilmReviewCard component render correctly`, () => {
  const tree = renderer.create(
      <FilmReviewCard
        review={review}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
