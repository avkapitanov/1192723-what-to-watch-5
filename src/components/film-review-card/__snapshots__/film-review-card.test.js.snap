// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmReviewCard component render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      I personally found this movie to be boring. Definitely one of the most boring movies I've ever seen.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Mollie
      </cite>
      <time
        className="review__date"
        dateTime="2020-10-30"
      >
        October 30, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    3.3
  </div>
</div>
`;
